package.json

```diff
@@ -34,8 +35,10 @@
   "devDependencies": {
     "@seneca/repl": "4.0.1",
     "@voxgig/model": "voxgig/model",
+    "electron": "21.1.0",
     "faker": "5.5.3",
     "jasmine": "3.8.0",
+    "json-dom-viewer": "^0.0.29",
```

Comment:
Please freeze the version. To install an npm package with a frozen version, you can use the `-E` option (means "exact version"), like so:
```
npm i json-dom-viewer -E
```

Reply:
Done